config/vufind/config.ini

```diff
@@ -2038,6 +2038,8 @@ HMACkey = mySuperSecretValue
 ; VUFIND_CACHE_DIR (see httpd-vufind.conf). The default location is inside the
 ; local settings directory.
 [Cache]
+; Uncomment the following line to disable all caching
```

Comment:
Right now, I think this is only disabling file-based caching, since there's only a check for it in createFileCache and not the other cache creation methods.

Reply:
Which other cache creation methods do you mean exactly?

As far as i could see, the VuFind\Cache\Manager only supports two creation methods: createFileCache (which is already part of the change), and createNoCache (which obviously doesn't need any options since it will create a BlackHole).

Or do you mean other parts of the code that do not use the Cache Manager at all? But that also means => if they don't use it, some special settings might have no effect at all, no matter if it's file based or not (if they even use the default [Cache] section).

Reply:
Sorry, @mtrojan-ub, there used to be another cache creation method using APC, but that was deprecated and removed, and I forgot that we only had file caching left -- I should have looked more closely at the code before complaining. :-)